tests/xlsx_export.test.ts

```diff
@@ -958,6 +959,58 @@ describe("Test XLSX export", () => {
     });
   });
 
+  describe("Images", () => {
+    const getModelData = () => ({
+      sheets: [
+        {
+          name: "Sheet1",
+          // Need a sheet big enough to contain the image completely.
```

Comment:
what happens if the sheet is not big enough ? does it crash ?

Reply:
> what happens if the sheet is not big enough ? does it crash ?

@rrahir It won't crash and will be exported as usual. I added a test for it and deleted this ambiguous comment.